lua/lspimport/init.lua

```diff
@@ -38,9 +37,7 @@ end
 ---@param unresolved_import string
 ---@return table[]
 local get_auto_import_complete_items = function(server, result, unresolved_import)
-    -- TODO: use vim.lsp.util.text_document_completion_list_to_complete_items
-    -- once stabilized on nvim API.
-    local items = require("vim.lsp._completion")._lsp_to_complete_items(result, unresolved_import)
+    local items = require("vim.lsp.util").text_document_completion_list_to_complete_items(result, unresolved_import)
```

Comment:
let's still leave a TODO to use a different function once this get deprecated, because _it is_ marked deprecated in 0.10